src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -849,14 +872,24 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
             var installedPackages = InstalledPackagesManager.InstalledPackages;
             var installed = installedPackages.FirstOrDefault(p => p.Id == package.Id);
 
-            EditorGUILayout.BeginHorizontal();
+            // if we are on the update tab, we do not want to show packages that have no updates if we're showing updates; similarly, we do not
+            // show packages that are on the lowest possible version if we're showing downgrades
+            if (currentTab == NugetWindowTab.UpdatesTab &&
+                installed != null &&
```

Comment:
```suggestion
                installed != null &&
                package.Versions.Count >= 1 &&
```
It shouldn't be possible because the list should at least contain the package version itself, but I would still check it so this doesn't throw a exception.